src/coreclr/gc/gc.cpp

```diff
@@ -30555,7 +30702,8 @@ heap_segment* gc_heap::get_new_region (int gen_number, size_t size)
         heap_segment_next (generation_tail_region (gen)) = new_region;
         generation_tail_region (gen) = new_region;
 
-        verify_regions (gen_number, false, settings.concurrent);
+        size_t unused_total_committed;
+        verify_regions (gen_number, false, settings.concurrent, unused_total_committed);
     }
```

Comment:
could get rid of this unused var by checking for null in `verify_regions` (change to a pointer type)